content/millennium/r4/other/health-cards.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Health Cards | R4 API
+---
+
+# Health-Cards
+
+* TOC
+{:toc}
+
+## Overview
+
+A Health Card is a verifiable, digital artifact that represents a limited set of information about an individual, relevant to a specific health status. The card can be presented in digital and physical form. A QR code representing the card can be shared from a device or a piece of paper. Information in the health card can be used to determine the authenticity of the artifact and its issuer. The goal is to provide individuals with the choice to share limited health information in a portable and secure manner.
+
+Health cards are designed in conformance with the emerging [SMART Health Card framework](https://smarthealth.cards) and [HL7 FHIR Vaccination and Testing IG](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/).
+
+The returned FHIR Parameter resource adhere to the [FHIR $health-cards-issue Operation Specification](https://spec.smarthealth.cards/#via-fhir-health-cards-issue-operation).
+
+The FHIR payloads encompassed in the Health Card adhere to the Data Minimization profiles defined by [HL7 FHIR Vaccination and Testing IG](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/).
+
+The following [FHIR Patient](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-dm.html#tab-ms) payload fields in the Health Card are returned if valued:
+
+* name (Only one name is returned)
+  * family
+  * given
+* birthDate
+
+The following [FHIR Immunization](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-immunization-dm.html#tab-ms) payload fields in the Health Card are returned if valued:
+
+* status
+* vaccineCode (Only supported codes for the credentialType are returned)
+* patient
+* occurrenceDateTime
+* lotNumber
+* performer (Only one Organization Reference is returned)
+
+## $health-cards-issue 
+
+Issue a  Health Card for a patient that meet the supplied request:
+
+    POST /Patient/{Patient.id}/$health-cards-issue
```

Comment:
Crud - I forgot to fix this. My bad.

I think the `{Patient.id}` should be `:id`:

```
POST /Patient/:id/$health-cards-issue
```

Reply:
Addressed in https://github.com/cerner/fhir.cerner.com/pull/659/commits/7b1c07475334668f659482c42b775168f4ef2cc5